untrusted/urts_sys/src/lib.rs

```diff
@@ -3,9 +3,11 @@
 //
 #![allow(non_camel_case_types, non_snake_case, non_upper_case_globals)]
 
+//TODO should report be propagated here or should callers talk to types directly?
```

Comment:
Note this todo, this could stay for a follow on.
I think now or in an immediate follow on may be time to create a "common" types crate.  
Initially I was thinking it would house the types that are used on both sides of the enclave interface, but I'm thinking it may be better to keep the 2 sides of the enclave separate at the crate level.
However "common" without guiding rules quickly blows up.